app/views/directives/annotations.html

```diff
@@ -1,10 +1,10 @@
-<p>
+<p ng-class="{'mar-bottom-xl': !expandAnnotations}">
```

Comment:
I don't mind the default margin, and it would save us some special case classes, but I'm okay with whatever you and @rhamilto think